STYLE_GUIDE.md

```diff
@@ -0,0 +1,461 @@
+# Pony Standard Library Style Guide
+
+This is a document describing the preferred practices for writing code for the Pony standard library. Other projects are free to use the same style, but you are free to use any style in other projects. The following conventions are meant to ensure consistency and readability of Pony code. Any of the following rules may be broken in situations where the recommended practice would diminish readability.
+
+
+## Formatting
+
+### Line Length
+
+The maximum line length is 80 columns.
```

Comment:
Outside of the stdlib, that's fine of course. For the stdlib, 80 works _everywhere_ without line wrapping problems. This is very useful on some displays - and even more often when I want to display two files side by side on a laptop :)